src/workerd/server/workerd.c++

```diff
@@ -807,7 +808,24 @@ class CliMain final: public SchemaFileImpl::ErrorReporter {
         context, getVersionString(), "Outputs the package lock file used by Pyodide.");
     return builder
         .callAfterParsing([]() -> kj::MainBuilder::Validity {
-      printf("%s\n", PYODIDE_LOCK->cStr());
+      static const PythonConfig config{
+        .packageDiskCacheRoot = kj::none,
+        .pyodideDiskCacheRoot = kj::none,
```

Comment:
Shouldn't this respect the `pyodide-bundle-disk-cache-dir` cli option?

Reply:
Possibly, but we can do so later. I just wanted to get rid of the use of the embedded PYODIDE_LOCK. I'll note this in the EW ticket above.